cfgov/unprocessed/apps/filing-instruction-guide/js/fig-search.js

```diff
@@ -83,4 +90,19 @@ const onFollow = (event) => {
   }
 };
 
+/**
+ * Event listener that's fired after a user searches for something.
+ * We're reporting the user's search terms to Google Analytics.
+ *
+ * @param {string} query - Search term that was submitted
```

Comment:
Trailing period `.` if you want to feel you've completed the last touch on a comment masterpiece 🎨